src/components/modal/CdrModal.vue

```diff
@@ -204,7 +205,68 @@ const addHandlers = () => {
   document.addEventListener('keydown', handleKeyDown);
 };
 
+/**
+ * Show / hide nodes on open / close
+ * backgroundContentNodes = nodes to target
+ */
+const backgroundContentNodes: Array<HTMLElement> = [];
+let backgroundNodesDiscovered = false;
+
+/**
+ * findBackgroundContentNodes
+ * When modal opens for the first time, find the content that should
+ * be hidden when it opens, and store it in local state
+ */
+const findBackgroundContentNodes = () => {
+  // initial selector is complex, breaking it down
+  const notSelectors = [
+    'body > *',
+    ':not(script)',
+    ':not(style)',
+    ':not([aria-hidden=true])',
```

Comment:
Prevents us from accidentally removing `aria-hidden="true"` from something that was set that way on page load.